zng/docs/spec.md

```diff
@@ -1,12 +1,15 @@
 # ZNG Specification
 
-> Note: This specification is ALPHA and a work in progress.
-> Zq's implementation of ZNG is tracking this spec and as it changes,
+> ### Note: This specification is ALPHA and a work in progress.
```

Comment:
By using `###` to make this into a header, I can link directly to it from within this doc or the "Zeek compatibility" one. I'd previously been linking to the "ZNG Specification" header since the alpha notice was already effectively at the top, but I figured it doesn't hurt to be explicit.